sdk/typescript/test/e2e/utils/setup.ts

```diff
@@ -106,5 +106,5 @@ export async function publishPackage(
   const publishEvent = getEvents(publishTxn).filter(
     (e: any) => 'publish' in e
   )[0];
-  return publishEvent.publish.packageId.replace(/^0x(0+)/, '$1');
+  return publishEvent.publish.packageId.replace(/^0x(0+)/, '0x');
```

Comment:
Thanks for fixing this! I made a typo here, should be `replace(/(^0x)(0+)/, '0x');` instead of `replace(/^0x(0+)/, '0x');`. Your change here is clearer